src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2954,14 +2971,14 @@
           }
         },
 
-        "artifactIndices": {
-          "description": "The indices within the run artifacts array of the artifact objects associated with the tool component.",
+        "artifacts": {
```

Comment:
>artifacts [](start = 9, length = 9)

we got this wrong. the type here s/be artifactLocation, not artifact. the name of the property as 'artifacts' is correct. though.